samples/bluetooth/ipsp/prj_zep1656_disco_l475_iot1.conf

```diff
@@ -0,0 +1,32 @@
+# CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL is not set
+CONFIG_CLOCK_STM32_SYSCLK_SRC_MSI=y
```

Comment:
I wonder why this is not in the board configuration? Everything else seems to be the same as in prj.conf so if the board sets it we wouldn't need a specific prj.conf.

Reply:
In general, we don't need to use this clock, for example with beacon sample it's not mandatory to change the clock but with ipsp sample we have seen some data lost during bluetooth communication and is why we have added this configuration specifically for this sample.